.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+# Related Issue
+- Issue goes here with URL/Number
+
+# Proposed Changes
```

Comment:
For this section, I would copy the language from the PR template in the main `opencue` repo.

https://raw.githubusercontent.com/AcademySoftwareFoundation/OpenCue/master/.github/PULL_REQUEST_TEMPLATE.md

Reply:
Did you mean I have to add whole issue section such as "One line for each Issue. If merging this Pull Request should auto-close the Issue on that line,
preface the Issue URL/Number with "Fixes". Every Pull Request should have at least one Issue
linked."

Reply:
I would use this whole section from the main repo's template:

```
**Link the Issue(s) this Pull Request is related to.**
One line for each Issue. If merging this Pull Request should auto-close the Issue on that line,
preface the Issue URL/Number with "Fixes". Every Pull Request should have at least one Issue
linked.

**Summarize your change.**
If it's a simple change, a sentence or two is probably sufficient. If it's a complex change,
adding a list of the three or four main points can be helpful.

It's also helpful to describe **why** you're making this change.
```

That text can take the place of the "Related Issue" and "Proposed Changes" sections you have here.

It will be good to keep things consistent between repos.